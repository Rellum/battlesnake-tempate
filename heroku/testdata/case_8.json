{
  "game": {
    "id": "04d4d7e1-f9cb-4ad2-bfd7-bf2d929e0734"
  },
  "turn": 316,
  "board": {
    "width": 11,
    "height": 11,
    "snakes": [
      {
        "id": "gs_XkfD93WMKKjmTMrvPxWTMDDK",
        "name": "Hadyn",
        "body": [
          {
            "x": 4,
            "y": 10
          },
          {
            "x": 4,
            "y": 9
          },
          {
            "x": 4,
            "y": 8
          },
          {
            "x": 4,
            "y": 7
          },
          {
            "x": 3,
            "y": 7
          },
          {
            "x": 3,
            "y": 6
          },
          {
            "x": 4,
            "y": 6
          },
          {
            "x": 4,
            "y": 5
          },
          {
            "x": 3,
            "y": 5
          },
          {
            "x": 3,
            "y": 4
          },
          {
            "x": 4,
            "y": 4
          },
          {
            "x": 5,
            "y": 4
          },
          {
            "x": 6,
            "y": 4
          },
          {
            "x": 7,
            "y": 4
          },
          {
            "x": 8,
            "y": 4
          },
          {
            "x": 9,
            "y": 4
          },
          {
            "x": 10,
            "y": 4
          },
          {
            "x": 10,
            "y": 3
          },
          {
            "x": 9,
            "y": 3
          },
          {
            "x": 8,
            "y": 3
          },
          {
            "x": 7,
            "y": 3
          },
          {
            "x": 6,
            "y": 3
          },
          {
            "x": 5,
            "y": 3
          },
          {
            "x": 4,
            "y": 3
          },
          {
            "x": 3,
            "y": 3
          },
          {
            "x": 2,
            "y": 3
          },
          {
            "x": 2,
            "y": 4
          },
          {
            "x": 2,
            "y": 5
          },
          {
            "x": 2,
            "y": 6
          },
          {
            "x": 2,
            "y": 7
          },
          {
            "x": 2,
            "y": 8
          },
          {
            "x": 2,
            "y": 9
          },
          {
            "x": 2,
            "y": 10
          },
          {
            "x": 1,
            "y": 10
          },
          {
            "x": 0,
            "y": 10
          },
          {
            "x": 0,
            "y": 9
          },
          {
            "x": 0,
            "y": 8
          },
          {
            "x": 0,
            "y": 7
          },
          {
            "x": 0,
            "y": 6
          },
          {
            "x": 0,
            "y": 5
          },
          {
            "x": 0,
            "y": 4
          },
          {
            "x": 0,
            "y": 3
          },
          {
            "x": 0,
            "y": 2
          },
          {
            "x": 0,
            "y": 1
          }
        ],
        "health": 98
      }
    ],
    "food": [
      {
        "x": 5,
        "y": 5
      },
      {
        "x": 1,
        "y": 7
      },
      {
        "x": 1,
        "y": 8
      },
      {
        "x": 1,
        "y": 6
      },
      {
        "x": 1,
        "y": 0
      },
      {
        "x": 5,
        "y": 6
      },
      {
        "x": 6,
        "y": 10
      },
      {
        "x": 6,
        "y": 2
      },
      {
        "x": 5,
        "y": 8
      },
      {
        "x": 9,
        "y": 8
      }
    ]
  },
  "you": {
    "id": "gs_XkfD93WMKKjmTMrvPxWTMDDK",
    "name": "Hadyn",
    "body": [
      {
        "x": 4,
        "y": 10
      },
      {
        "x": 4,
        "y": 9
      },
      {
        "x": 4,
        "y": 8
      },
      {
        "x": 4,
        "y": 7
      },
      {
        "x": 3,
        "y": 7
      },
      {
        "x": 3,
        "y": 6
      },
      {
        "x": 4,
        "y": 6
      },
      {
        "x": 4,
        "y": 5
      },
      {
        "x": 3,
        "y": 5
      },
      {
        "x": 3,
        "y": 4
      },
      {
        "x": 4,
        "y": 4
      },
      {
        "x": 5,
        "y": 4
      },
      {
        "x": 6,
        "y": 4
      },
      {
        "x": 7,
        "y": 4
      },
      {
        "x": 8,
        "y": 4
      },
      {
        "x": 9,
        "y": 4
      },
      {
        "x": 10,
        "y": 4
      },
      {
        "x": 10,
        "y": 3
      },
      {
        "x": 9,
        "y": 3
      },
      {
        "x": 8,
        "y": 3
      },
      {
        "x": 7,
        "y": 3
      },
      {
        "x": 6,
        "y": 3
      },
      {
        "x": 5,
        "y": 3
      },
      {
        "x": 4,
        "y": 3
      },
      {
        "x": 3,
        "y": 3
      },
      {
        "x": 2,
        "y": 3
      },
      {
        "x": 2,
        "y": 4
      },
      {
        "x": 2,
        "y": 5
      },
      {
        "x": 2,
        "y": 6
      },
      {
        "x": 2,
        "y": 7
      },
      {
        "x": 2,
        "y": 8
      },
      {
        "x": 2,
        "y": 9
      },
      {
        "x": 2,
        "y": 10
      },
      {
        "x": 1,
        "y": 10
      },
      {
        "x": 0,
        "y": 10
      },
      {
        "x": 0,
        "y": 9
      },
      {
        "x": 0,
        "y": 8
      },
      {
        "x": 0,
        "y": 7
      },
      {
        "x": 0,
        "y": 6
      },
      {
        "x": 0,
        "y": 5
      },
      {
        "x": 0,
        "y": 4
      },
      {
        "x": 0,
        "y": 3
      },
      {
        "x": 0,
        "y": 2
      },
      {
        "x": 0,
        "y": 1
      }
    ],
    "health": 98
  }
}