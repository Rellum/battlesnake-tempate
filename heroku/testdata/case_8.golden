|yyy Y f    |
|y y y      |
|yfy yf   f |
|yfyyy      |
|yfyyyf     |
|y yyyf     |
|y yyyyyyyyy|
|y yyyyyyyyy|
|y     f    |
|y          |
| f         |
